awd/config.py

```diff
@@ -58,3 +61,12 @@
     LOG_SOURCE_EMAIL = os.getenv("LOG_SOURCE_EMAIL")
     LOG_RECIPIENT_EMAIL = os.getenv("LOG_RECIPIENT_EMAIL")
     RETRY_THRESHOLD = os.getenv("RETRY_THRESHOLD")
+    SENTRY_DSN = os.getenv("SENTRY_DSN")
+
+
+def check_sentry():
+    if SENTRY_DSN:
```

Comment:
What is the intent behind this? If SENTRY_DSN is absent, line 71 duly reports that.

Reply:
This is called when you need to check that the actual connection to Sentry is receiving exception. It is just being stored for those situations, this is happening in the `alma-scripts` and `dspace-submission-service`repos as well

Reply:
It is not regularly called, it is there in for troubleshooting purposes, see the last comment here https://github.com/MITLibraries/alma-scripts/pull/33

Reply:
Removed